README.md

```diff
@@ -11,7 +11,8 @@ Read things here before you start.
 - [Distributed systems theory for the distributed engineer](http://the-paper-trail.org/blog/distributed-systems-theory-for-the-distributed-systems-engineer/), most of the papers/books in the blog might reappear in this list again. Still a good BFS approach to distributed systems.
 - [FLP Impossibility Result (paper)](https://groups.csail.mit.edu/tds/papers/Lynch/jacm85.pdf), an easier [blog post](http://the-paper-trail.org/blog/a-brief-tour-of-flp-impossibility/) to follow along
 - [An Introduction to Distributed Systems](https://github.com/aphyr/distsys-class) @aphyr's excellent introduction to distributed systems 
-
+- [A Guide to Consistent Hashing](https://www.toptal.com/big-data/consistent-hashing) Complete guide on consistent hashing
```

Comment:
Can you move this under storage and databases, also there is an extraneous `-` character in the next newline